erp/management/commands/remove_duplicates.py

```diff
@@ -0,0 +1,114 @@
+            elif 75 < distance_diff <= 500:
+                try:
+                    is_duplicate = check_for_automatic_merge([erp, potential_duplicate])
+                    if is_duplicate:
+                        duplicates.append(potential_duplicate.pk)
+                except NeedsManualInspectionException:
+                    message = f"{erp.pk};{potential_duplicate.pk};Need manual check for ERP {erp.nom} with {potential_duplicate.nom}"
+                    self.for_manual_review.append(message)
+                except NotDuplicatesException:
+                    pass
+
+        return Erp.objects.filter(pk__in=duplicates).select_related("accessibilite")
+
+    def _check_for_asp_id(self, erp, duplicates):
+        ids = set([e.asp_id for e in duplicates if e.asp_id])
+        if len(ids) == 1:
+            erp.asp_id = list(ids)[0]
+        elif ids:
+            self.stderr.write(f"Can't find the correct ASP ID for {erp} with {duplicates}")
+            self.stderr.write(f"Found possible ids {ids}")
+
+    def _check_for_street_number(self, erp, duplicates):
+        numbers = set([e.numero for e in duplicates if e.asp_id])
+        if len(numbers) == 1:
+            erp.numero = list(numbers)[0]
+        elif numbers:
+            self.stderr.write(f"Can't find the correct Numero de voie for {erp} with {duplicates}")
+            self.stderr.write(f"Found possible ids {numbers}")
+
+    def _merge_and_delete(self, erp, duplicates):
+        if len(duplicates) > 1:
+            self.unhandled += len(duplicates)
+            self.stderr.write(f"{len(duplicates)} ERPs found - Need to improve merge strategy in this case")
+            return
+
+        print("Merge and delete 1 ERP")
+        if self.should_write:
+            merge_accessibility_with(erp, duplicates[0])
+            print(f"Will delete {duplicates[0]}")
+            duplicates[0].delete()
+        self.to_delete += 1
+
+    def handle(self, *args, **options):
+        self.should_write = options["write"]
+        queryset = Erp.objects.filter(published=True).order_by("created_at")
+        for erp in queryset.iterator():
+            try:
+                erp.refresh_from_db()
+            except Erp.DoesNotExist:
+                continue
+
+            duplicates = self._get_duplicates(erp)
+            if not duplicates:
+                continue
+
+            main_erp, duplicates = find_main_erp_and_duplicates([erp, *duplicates])
+            if not duplicates:
+                continue
+            print(main_erp, duplicates)
+            self._check_for_asp_id(main_erp, duplicates)
+            self._check_for_street_number(main_erp, duplicates)
+
+            if main_erp.shares_same_accessibility_data_with(duplicates):
+                self.to_delete += len(duplicates)
+                if self.should_write:
+                    print(f"Will delete {duplicates}")
+                    duplicates.delete()
+                erp.save()
```

Comment:
here we are saving even if we are in dry run mode, same on the above else, is it expected?